package.json

```diff
@@ -7,10 +7,11 @@
     "jfk": "./bin/jfk.js"
   },
   "scripts": {
-    "prepare": "./scripts/download.js",
+    "prepare": "node ./scripts/download.js",
```

Comment:
Why? To support windows build?

Reply:
In other words, what if the default script interpreter is not `node`?

Reply:
https://github.com/b1f6c1c4/IPA-JFK/blob/master/scripts/download.js#L1

Reply:
The first line was not recognized in WIndows